tasks/seq/example/func_tests/main.cpp

```diff
@@ -118,3 +119,33 @@ TEST(Sequential, Test_Sum_100) {
   test_task_sequential.post_processing();
   ASSERT_EQ(count, out[0]);
 }
+
+TEST(Sequential, Test_Sum_100_From_File) {
```

Comment:
Shall we add this for all (MPI/OMP/TBB/...)?

Reply:
I agree, I will correct

Reply:
done